### PR TITLE
Unit 1 UX Team Intern Onboarding Project 

### PR DESCRIPTION
**Pull request by frankiebarrios**
## What It Does
This pull request is set to merge repo-setup into master using the pull request template. 

This work is taking place to complete unit 1 of the ux-onboarding-project.

**Implementation:**
- Created the repo.
- Made the necessary updates (included adding both pull request and issue templates in a new directory within the repo).
- Created the pull request utilizing the pull request template (vs the issue template as stated in 1.1-project-setup.md, not sure if this is one of the sections that may need updated as Eli requested us to look out for in Slack?).
- Submitted the pull request.

[closes 1.1: Project and Github Setup](https://github.com/Banno/ux-intern-onboarding/blob/master/issues/unit-1/1.1-project-setup.md)

## How To Test

N/A

## Notes/Caveats

None
## Checklist

Under penalty of public shaming, I avow that I:
**Many items in list N/A** 
- [X] Reviewed the diff to look for typos, whitespace errors, and console.log() statements
- [X] Verified that there are no compiler or linter errors or warnings
- [X] Verified the build in production(optimized) mode
- [X] Updated the docs, if necessary
- [X] Included the appropriate labels
- [X] Referenced applicable issues (i.e. Closes #XXXX, Fixes #XXXX)

Browsers tested:
**Many items in list N/A** 
- [X] Chrome
- [X] Firefox
- [X] Safari
- [X] Edge
- [X] Internet Explorer

## Dependencies
1. Install Node and npm
2. Install Yarn

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "frankie-barrios-ux-iop-1",
+  "version": "1.0.0",
+  "description": "UX Team Intern Onboarding Project",
+  "main": "index.js",
```
Comment (kddrake) [inline package.json:5]: won't need to define `main`, for the time being anyway.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  "scripts": {
```
Comment (kddrake) [inline package.json:6]: same as `main`, won't need to define any scripts for the time being.

**Review by kddrake — APPROVED** [review]
👍 looks good to me. have a few comments but nothing that _needs_ changed

File: package.json (hunk 1 above)
Comment (frankiebarrios) [reply package.json:5]: Gotcha, thanks.

**Pull request merged**